graylog2-web-interface/src/components/common/IconButton.tsx

```diff
@@ -49,7 +49,7 @@ type Props = {
   title: string,
   onClick?: () => void,
   className?: string,
-  name: string,
+  name?: string,
```

Comment:
Why did you typed this prop as optional?

Reply:
It seemed to me as if it was not specified everytime the component is used, but it looks like I was wrong. Changed that!